### PR TITLE
novacustom_ns5x_7x/releases.md: add info about ec transition

### PR DESCRIPTION
**Pull request by sulewskiprzemyslaw**
Signed-off-by: Przemyslaw Sulewski <przemyslaw.sulewski@3mdeb.com>

### CODE REVIEW

File: docs/variants/novacustom_ns5x_7x/releases.md [hunk 1]
```diff
@@ -54,6 +54,15 @@ Test results for this platform can be found
 
 See how to verify signatures on [this video](https://asciinema.org/a/518379)
 
+### EC firmware transition
+
+Note that, version 1.3.0 of Dasharo firmware works correctly only with the
```
Comment (pietrushnic) [inline docs/variants/novacustom_ns5x_7x/releases.md:59]:
```suggestion
Note that, version 1.3.0 of Dasharo firmware works correctly **only** with the
```

File: docs/variants/novacustom_ns5x_7x/releases.md [hunk 2]
```diff
@@ -54,6 +54,15 @@ Test results for this platform can be found
...
+open EC firmware.
+
+Transition to the open EC firmware might be done:
+
+- by using [EC flashing kit](ec_recovery.md).
```
Comment (pietrushnic) [inline docs/variants/novacustom_ns5x_7x/releases.md:64]: @sulewskiprzemyslaw of course we should add some disclaimer that flashing new firmware without EC may lead to bricking system and we are not responsible for such demage.

File: docs/variants/novacustom_ns5x_7x/releases.md (hunk 1 above)
Comment (sulewskiprzemyslaw) [reply docs/variants/novacustom_ns5x_7x/releases.md]: added

File: docs/variants/novacustom_ns5x_7x/releases.md (hunk 2 above)
Comment (sulewskiprzemyslaw) [reply docs/variants/novacustom_ns5x_7x/releases.md:64]: added

File: docs/variants/novacustom_ns5x_7x/releases.md [hunk 3]
```diff
@@ -54,6 +54,21 @@ Test results for this platform can be found
...
+
+EC transition procedure should be performed immediately after flashing the new
+version of the firmware.
+
+Usage firmware v1.3.0 without open EC firmware may cause in bricking the
```
Comment (miczyg1) [inline docs/variants/novacustom_ns5x_7x/releases.md:69]:
```suggestion
Usage firmware v1.3.0 without open EC firmware may cause bricking the
```

File: docs/variants/novacustom_ns5x_7x/releases.md [hunk 4]
```diff
@@ -54,6 +54,21 @@ Test results for this platform can be found
...
+
+Transition to the open EC firmware might be done:
+
+- by using [EC flashing kit](ec_recovery.md).
+
+EC transition procedure should be performed immediately after flashing the new
+version of the firmware. 
+
+Usage firmware v1.3.0 without open EC firmware may cause in bricking the
+system. We are not responsible for such a damage.
```
Comment (pietrushnic) [inline docs/variants/novacustom_ns5x_7x/releases.md:70]:
```suggestion
Using Dasharo firmware v1.3.0 without open EC firmware may cause bricking of the system, and we are not responsible for such damage.
```

Please wrap accordingly.

File: docs/variants/novacustom_ns5x_7x/releases.md [hunk 5]
```diff
@@ -54,6 +54,21 @@ Test results for this platform can be found
...
 See how to verify signatures on [this video](https://asciinema.org/a/518379)
 
+### EC firmware transition
+
+Note that, version 1.3.0 of Dasharo firmware works correctly **only** with the
+open EC firmware.
+
+Transition to the open EC firmware might be done:
+
+- by using [EC flashing kit](ec_recovery.md).
```
Comment (miczyg1) [inline docs/variants/novacustom_ns5x_7x/releases.md:64]: What about DTS?

**Review by miczyg1 — CHANGES_REQUESTED** [review]

File: docs/variants/novacustom_ns5x_7x/releases.md (hunk 4 above)
Comment (sulewskiprzemyslaw) [reply docs/variants/novacustom_ns5x_7x/releases.md]: added

**Review by miczyg1 — APPROVED** [review]

**Pull request merged**